### PR TITLE
chapter 6: Modeling users

### PR DESCRIPTION
**Pull request by dinhmuoi97**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  validates :name, presence: true, length: {maximun: Settings.max_user}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true
+  , length: {maximun: Settings.max_email},
```
Comment (dodat1905) [inline app/models/user.rb]: indent a

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'users/new'
```
Comment (dodat1905) [inline config/routes.rb]: dùng nháy kép a nhé

**Comment by dodat1905** [comment]
commented

File: config/routes.rb (hunk 2 above)
Comment (dinhmuoi97) [reply config/routes.rb]: yep

**Comment by hungnh103** [comment]
commented
em merge pull nhé ;)

**Pull request merged**